src/orion/core/__init__.py

```diff
@@ -107,7 +107,7 @@ def define_database_config(config):
     database_config.add_option(
         "type",
         option_type=str,
-        default="MongoDB",
+        default="PickledDB",
         env_var="ORION_DB_TYPE",
         help=(
             "Type of database. Builtin backends are ``mongodb``, "
```

Comment:
You will need to change the host below as well. You should set it to default host for pickleddb: https://github.com/Epistimio/orion/blob/develop/src/orion/core/io/database/pickleddb.py#L25

Reply:
Ah, right. Shouldn't the default here depend on the actual value of the `type` option? I mean, if `type` is set to `mongodb` then `host` should be `localhost` by default, not PickledDB's `DEFAULT_HOST`.

Reply:
Ideally yes, but `Configuration` does not support conditional defaults like this for the moment. :(

Reply:
Good idea!